File: query-event-service/src/main/java/com/thesaastech/tickket/service/event/web/rest/dto/Events.java
package com.thesaastech.tickket.service.event.web.rest.dto;

import java.time.LocalDateTime;
import java.util.UUID;

public class Events {

    private UUID eventId;
    private String type;
    private String title;
    private String description;
    private LocalDateTime dateTime;
    private String venueAddressLine;
    private String venueCity;
    private String venuePostcode;

}

File: cmd-event-service/src/main/kotlin/com/thesaastech/tickket/data/events/events.kt
package com.thesaastech.tickket.data.events

import java.time.Instant
import java.time.LocalDateTime
import java.util.*

data class EventCreated(
    val eventId: UUID, val userId: String, val type: String, val title: String,
    val description: String, val dataTime: LocalDateTime, val venueAddressLine: String, val venueCity: String,
    val venuePostcode: String, val status: String
)
